### PR TITLE
chore(PPDSC-2046): review PR template

### PR DESCRIPTION
**Pull request by evgenitsn**
[PPDSC-2046](https://nidigitalsolutions.jira.com/browse/PPDSC-2046)

**What**

1. Background - why this is needed
The PR template was a bit outdated.

2. What did you do
Updated the PR template

3. What does the reviewers should expect
- For the current changes I've investigated a bunch of other open source design systems PR template but most of them contained the same information that we currently have or less. So there are no new rules/checkboxes added to our template.

- I've removed some of our current checkboxes after a talk with @mutebg . Please review the changes and if you think we should bring back something or we should add a new one feel free to leave a comment. I am still new to the project so it's possible that I don't understand the reasoning behind each checkbox.

**I have done:**
 - [ ] Written unit tests against changes
 - [ ] Written accessibility tests against the component
 - [ ] Written functional tests against the component and/or NewsKit site
 - [x] Updated relevant documentation
 - [ ] Updated the release notes for the [next release version](https://nidigitalsolutions.jira.com/wiki/spaces/NPP/pages/2349695021/Release%2BNotes%2BCode%2B-%2BWeb%2BDocumentation)
 - [ ] Discussed with QA and/or design about any visual changes

**I have tested manually:**
 - [ ] Renders correctly on IE11
 - [ ] The feature's functionality is working as expected on other browsers such as Safari, Firefox and IE11
 - [ ] There are no accessibility errors reported by WAVE
 - [ ] The screen reader reads the expected text for the tabbable elements.
 - [ ] There are no new errors in the browser console coming from this PR.
 - [ ] When visual test is not added, it renders correctly on different browsers and mobile viewports (Safari, Firefox, small mobile viewport, tablet)
 - [ ] The Playground feature is working as expected

### CODE REVIEW

**Comment by newskit-engineering** [comment]
You can preview these changes on: 
  Documentation Site: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2046-review-pr-template 
  Storybook: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2046-review-pr-template/storybook

**Review by baburay23 — COMMENTED** [review]
hey @evgenitsn this looks good to me happy to approve if others have no feedback

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -19,26 +19,20 @@ DO CONSIDER any of the following are breaking changes to a consumer, this is by
...
- - [ ] Renders correctly on IE11
- - [ ] The feature's functionality is working as expected on other browsers such as Safari, Firefox and IE11
- - [ ] There are no accessibility errors reported by WAVE
- - [ ] The screen reader reads the expected text for the tabbable elements.
- - [ ] There are no new errors in the browser console coming from this PR.
- - [ ] When visual test is not added, it renders correctly on different browsers and mobile viewports (Safari, Firefox, small mobile viewport, tablet)
- - [ ] The Playground feature is working as expected
-
+- [ ] The feature's functionality is working as expected on Chrome, Firefox, Safari and Edge
+- [ ] The screen reader reads the expected text for the tabbable elements.
```
Comment (Vanals) [inline .github/PULL_REQUEST_TEMPLATE.md:32]:
```suggestion
- [ ] The screen reader reads and flows through the elements as expected.
```

I would change it to this! A bit more correct, as is not just for tabbable elements! 👍

**Comment by Vanals** [comment]
Wondering if is worth having a little huddle as mentioned in the ticket? One of the issue we had with the previous template was that no one was using it, so wondering is worth having everyone on the same page.

**Comment by evgenitsn** [comment]
> Wondering if is worth having a little huddle as mentioned in the ticket? One of the issue we had with the previous template was that no one was using it, so wondering is worth having everyone on the same page.

Yup, I am cool with that. How are we doing these? I am not sure what the best way to organise this.

**Review by mutebg — APPROVED** [review]

**Review by Vanals — APPROVED** [review]

**Pull request merged**